content/es/registrar/pr.md

```diff
@@ -3,15 +3,22 @@
 state_name = "Puerto Rico"
 state_abbreviation = "pr"
 is_state = "false"
-registration_type = "in-person"
+registration_type = "online"
 translationKey = "Puerto Rico"
 type = "register"
 hp_link = "https://ww2.ceepur.org/Home/Index?aspxerrorpath=/es-pr/Paginas/default.aspx"
+registration_link = "https://ere.ceepur.org/"
 more_info_link = "https://ww2.ceepur.org/es-pr/Paginas/cee-english.aspx"
 confirm_registration_link = "https://consulta.ceepur.org/"
+national_voter_registration_form_accepted = "false"
 hp_link_translated = "true"
 confirm_registration_link_translated = "true"
+registration_link_translated = "true"
+more_info_link_translated = "true"
 default_ip_deadline = "hasta 50 días antes del día de las elecciones"
+default_online_deadline = "hasta 50 días antes del día de las elecciones"
+registration_link_override = "https://ere.ceepur.org/"
+confirm_registration_link_override = "https://consulta.ceepur.org/"
```

Comment:
i saw that there are some states that have more added as far as links and overrides... is this just state specific?

Reply:
@clmedders Yes this is state specific! Some states offer more languages that have their own url.
In Puerto Rico's case, I added the links in the Spanish file so that (In English) is not printed. The default language for these links in Spanish.

Reply:
Yes those override links are unecessary because it is the same link as the fallback. The spanish override links should be removed unless there is an alternative language specific one:
https://ere.ceepur.org/
https://consulta.ceepur.org/

Reply:
Removed the 2 links that were present in the states.json file.